### PR TITLE
Add 2017 PyCharm Fundraiser banners

### PR DESCRIPTION
**Pull request by frankwiles**
Hi Everyone, 

We're going to be running the PyCharm promotion again starting today.  @timgraham can we get this merged and live asap? 

Going to announce this from the DjangoCon stage this morning.

### CODE REVIEW

File: djangoproject/templates/base.html [hunk 1]
```diff
@@ -43,7 +43,7 @@
   <body id="{% block sectionid %}generic{% endblock %}" class="{% block body_class %}{% endblock %}">
 
     {% include "includes/header.html" %}
-
+ -
```
Comment (timgraham) [inline djangoproject/templates/base.html]: Is this change intentional?

File: djangoproject/templates/fundraising/index.html [hunk 2]
```diff
@@ -5,8 +5,16 @@
 {% block layout_class %}full-width{% endblock %}
 
 {% block header %}
+  <div class="copy-banner copy-banner-black">
+    <div class="container">
+      <h1 class="visuallyhidden">Until July 20th, <a href="http://jb.gg/i-donate-to-django">get PyCharm at 30% off</a> All money goes to the Django Software Foundation</h1>
```
Comment (timgraham) [inline djangoproject/templates/fundraising/index.html]: August 28th

File: djangoproject/templates/base.html (hunk 1 above)
Comment (frankwiles) [inline djangoproject/templates/base.html]: Nope, fixing.

File: djangoproject/templates/fundraising/index.html (hunk 2 above)
Comment (frankwiles) [inline djangoproject/templates/fundraising/index.html]: Doh good catch thanks!

**Comment by frankwiles** [comment]
Ok @timgraham should be all fixed now.  Apologies was rushing to get this done before we announce!

**Comment by frankwiles** [comment]
Thanks!
